src/User.ts

```diff
@@ -27,18 +27,14 @@ import { Utils } from "./Utils";
 
 export class User {
 
-  displayableId: string;
-  name: string;
-  identityProvider: string;
+  idToken: IdToken;
```

Comment:
@davidmorissette We cannot define idToken as a public property as that will allow the developers to alter it.